From 3871428b928ec1043c41873484934b115083ece4 Mon Sep 17 00:00:00 2001
From: Rakesh Ranjan Jha <rakesh.r.jha@intel.com>
Date: Wed, 25 Nov 2015 11:26:52 +0530
Subject: [PATCH 11/12] Update the check for conditional control of V1P2A rail

Customer can sometimes update the SMBIOS table to change
board info so change the check such that it is executed
only for RVP

Change-Id: I61fb2a668f33f18ee6721fb588f2e70b824040b9
Signed-off-by: Rakesh Ranjan Jha <rakesh.r.jha@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-9935
Reviewed-on: https://android.intel.com:443/440423
---
 drivers/mfd/intel_soc_pmic_wc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mfd/intel_soc_pmic_wc.c b/drivers/mfd/intel_soc_pmic_wc.c
index b6e1e8e..15dacd7 100644
--- a/drivers/mfd/intel_soc_pmic_wc.c
+++ b/drivers/mfd/intel_soc_pmic_wc.c
@@ -907,7 +907,7 @@ static int whiskey_cove_init(void)
 	* and is required to be always enabled so skip for RVP
 	*/
 
-	if (board_name && !strcmp(board_name, "Cherry Trail FFD"))
+	if (board_name && strcmp(board_name, "Cherry Trail Tablet"))
 		wc_set_v1p2_pdata();
 
 	wc_set_v1p8_pdata();
-- 
1.9.1

